.eslintrc

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Same here, why not using the same configurations as in foreman?

Reply:
It's using Foreman's config now